packages/eas-cli/src/build/createContext.ts

```diff
@@ -43,13 +45,48 @@ export async function createBuildContextAsync<T extends Platform>({
   const workflow = await resolveWorkflowAsync(projectDir, platform);
   const accountId = findAccountByName(user.accounts, accountName)?.id;
 
-  const credentialsCtx = new CredentialsContext({
+  let credentialsCtxOptions = {
```

Comment:
Try avoiding using `let`. It's an easy refactor here.

Reply:
i've gotten rid of this code since we authenticate using env vars/ prompts now.